### ISSUE TITLE
Compression codec augmentation ?

### ISSUE THREAD

**Issue by tcourat** [wenet-e2e/wespeaker#364]
Hi,

Is there a way to randomly add a codec compression as a data augmentation when training speaker embeddings ? Is it already done in current pre-trained models ?

Things like Opus, MP3 etc.. but also telephony like a-law, mu-law etc.

Thanks.

**Comment by JiJiJiang** [comment]
No, we did not do any codec augmentation in our training data pipeline.
It would be very nice if you can contribute the codes with experimental results!
